Flow.Launcher.Test/Flow.Launcher.Test.csproj

```diff
@@ -10,6 +10,8 @@
     <ApplicationIcon />
     <StartupObject />
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
```

Comment:
These two properties are not necessary now that the ci fix is merged in.

Reply:
Yeah i will remove them.